### PR TITLE
Task/48/design create request view

### PR DESCRIPTION
**Pull request by nadamels**
Centered Create Request
Changed font 
Added close button

### CODE REVIEW

File: front-end/package-lock.json [hunk 1]
```diff
@@ -12580,6 +12581,14 @@
         "mkdirp": "bin/cmd.js"
       }
     },
+    "node_modules/modules": {
```
Comment (swostikpati) [inline front-end/package-lock.json:12584]: why is this added three times
image

File: front-end/src/components/student/CreateRequest.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    const handleDepartmentChange = (event) => {
+        setDepartment(event.target.value);
+    };
+
+    const handleFileChange = (event) => {
+        setSelectedFile(event.target.files[0]);
+    };
+
+    return (
+        <div id="createRequestComponent" className={`request-container ${isVisible ? 'visible' : ''}`}>
```
Comment (swostikpati) [inline front-end/src/components/student/CreateRequest.js:27]: give a css classnames here like create-request-component and then use it for styling in each one of the css classes

File: front-end/src/components/student/CreateRequest.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
+@import url("../../assets/styles.css");
+
+
```
Comment (swostikpati) [inline front-end/src/components/student/CreateRequest.css:3]: add .create-request-component to the main div and then add that before styling each class to ensure there aren't any class-collisions. Like this

.create-request-component .request-container
see the CreateRequest.js file for more info

File: front-end/src/components/student/CreateRequest.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            </div>
+
+            <div className="input-group file-group">
+                <div>
+                    <label className='file-label'>Attach File(s)</label>
+                </div>
+                <input className='file-select' type="file" id="fileInput" onChange={handleFileChange} />
+            </div>
+            <button type="submit" className='close-button' onClick={onClose}>Post Request</button>
+            <button className='x-button' onClick={onClose}>x</button>
```
Comment (swostikpati) [inline front-end/src/components/student/CreateRequest.js:83]: when the post requrst button is clicked, the information added by the user should be sent as a POST request to a mock backend API. Right now the information is just lost. Without this we won't be able to get the info at the backend. You might also consider using a form for this purpose. Check the login page implementation for more reference

**Comment by swostikpati** [comment]
Comments given in code

File: front-end/src/components/student/CreateRequest.js (hunk 2 above)
Comment (nadamels) [reply front-end/src/components/student/CreateRequest.js:27]: I just changed the classname in the top div "request-container" to "create-request-component-student"
